frontend/scripts/react-components/shared/tabs/tabs.component.jsx

```diff
@@ -14,35 +14,39 @@ function Tabs(props) {
     selectedTab,
     testId,
     getTabId,
-    color
+    color,
+    extraDropdown
   } = props;
   const isSelected = item => getTabId(item) === selectedTab;
   return (
     <div className="c-tabs">
       <div className="tabs-container">
-        {tabs.map((item, index) => (
-          <button
-            key={index}
-            className={cx('tab', {
-              '-selected': isSelected(item),
-              [color]: !!color
-            })}
-            data-key={item}
-            onClick={() => onSelectTab(item, index)}
-            data-test={`${testId}-item`}
-            disabled={isSelected(item)}
-          >
-            <Text
-              as="span"
-              color={isSelected(item) ? color : 'grey'}
-              weight="bold"
-              size="rg"
-              variant="mono"
+        <div className="tabs-options">
+          {tabs.map((item, index) => (
+            <button
+              key={index}
+              className={cx('tab', {
+                '-selected': isSelected(item),
+                [color]: !!color
+              })}
+              data-key={item}
+              onClick={() => onSelectTab(item, index)}
+              data-test={`${testId}-item`}
+              disabled={isSelected(item)}
             >
-              {itemTabRenderer ? itemTabRenderer(item, index) : item}
-            </Text>
-          </button>
-        ))}
+              <Text
+                as="span"
+                color={isSelected(item) ? color : 'grey'}
+                weight="bold"
+                size="rg"
+                variant="mono"
+              >
+                {itemTabRenderer ? itemTabRenderer(item, index) : item}
+              </Text>
+            </button>
+          ))}
+        </div>
+        {extraDropdown}
```

Comment:
I don't think this should be part of the tabs. The dropdown is a separate component that should be a sibling.